.github/workflows/agw.main_push.yml

```diff
@@ -2,7 +2,7 @@ name: merge-to-main
 on:
   push:
     branches:
-      - main
```

Comment:
it was discussed in[ this pr](https://github.com/canonical/charmed-magma-orchestrator/pull/159/files) that we don't want to keep a copy for each branch in the case of orc8r.
We could do the same here as well, right?

Reply:
just figured this might be problematic for version fixing, like in this case the version of tox :(

Reply:
Right... 
Branching out kinda creates a copy of every file in the repo anyway. I see a value of keeping CI config backwards compatible (if we wanted to back port some updates), but in any case, some manual work on every branch will be needed to apply any potential changes (like new CI steps). I also don't expect targets (like the `1.6` instead of `main` here) to change in the branch...

Reply:
approved based on our discussion today to keep the files branch specific.